src/gui/folderman.cpp

```diff
@@ -1507,9 +1539,14 @@ void FolderMan::editFileLocally(const QString &userId, const QString &relPath, c
         return;
     }
 
-    const auto checkTokenForEditLocally = new SimpleApiJob(accountFound->account(), QStringLiteral("/ocs/v2.php/apps/files/api/v1/openlocaleditor/%1").arg(token));
+    const QString encodedToken(QUrl::toPercentEncoding(token));   // Sanitise the token
```

Comment:
do not you miss a call to `QString::fromUtf8` ?

Reply:
It was working fine just with the constructor, but I have replaced it with a call to `fromUtf8` just in case